### ISSUE TITLE
unable to add dashboards in grafana

### ISSUE THREAD

**Issue by aliartiza75** [openshift/origin-web-console#3153]
Hi

I have deployed OCP cluster as a service on Azure. Once everything is deployed I found out that a project exists named as `openshift-monitoring`. It has grafana, Prometheus and node exporter in it. But the issue is that when I tried to add additional dashboards in it, I was not able to do it because it is a read-only instance for users.

Therefore I have deployed the same services(grafana and Prometheus) in another project and want to get the metrics of the node exporter running in the `openshift-monitoring` project.

I have created a prometheus service monitor in the new project for the node-exporter service in the `openshift-monitoring` project. 

Currently, I am not successful to get the metrics. Is there some other way to do it. 

### Version
openshift version 3.11

**Comment by spadgett** [comment]
Hi, this is probably best asked in https://github.com/openshift/cluster-monitoring-operator/

@s-urbaniak can you help?

**Comment by aliartiza75** [comment]
@spadgett thank you for the response. I have resolved this issue by running the node exporter in the new project and I assigned an empty string in the node selector section of the project. Before assigned the empty string the node exporter was running only in the compute nodes but this above change the node exporter now runs in all nodes.

OCP cluster specs:

3 master nodes
3 infra nodes
2 compute nodes
